simpeg/typing/__init__.py

```diff
@@ -27,8 +30,7 @@
     np.random.BitGenerator,
     np.random.Generator,
 ]
-
-RandomSeed.__doc__ = """
```

Comment:
So as it turns out, we shouldn't assign to `__doc__` here, as you are actually setting the docstring on `typing.Union`, because that is what `RandomSeed` is equal to.

Reply:
Good to know! Nice catch!